--- origsrc/texlive-20110705-source/texk/ptexenc/Makefile.am	2011-01-12 04:43:40.000000000 -0600
+++ src/texlive-20110705-source/texk/ptexenc/Makefile.am	2012-01-17 15:02:38.059249300 -0600
@@ -18,6 +18,8 @@ libptexenc_la_CPPFLAGS = -DMAKE_PTENC_DL
 
 libptexenc_la_LDFLAGS = -no-undefined -version-info $(PTEXENC_LT_VERSINFO)
 
+libptexenc_la_LIBADD = $(KPATHSEA_LIBS)
+
 # Note: jisx0208.h is a copy of ../../libs/gd/gd-2.0.35/jisx0208.h.
 #
 libptexenc_la_SOURCES = \
--- origsrc/texlive-20110705-source/texk/xdvipdfmx/src/Makefile.am	2011-03-10 03:44:56.000000000 -0600
+++ src/texlive-20110705-source/texk/xdvipdfmx/src/Makefile.am	2012-01-17 19:04:34.119738700 -0600
@@ -36,7 +36,7 @@ xdvipdfmx_SOURCES = \
 
 xdvipdfmx_DEPENDENCIES = $(ZLIB_DEPEND) $(LIBPNG_DEPEND) $(LIBPNG_FREETYPE2) $(KPATHSEA_DEPEND)
 
-LDADD = $(KPATHSEA_LIBS) $(FREETYPE2_LIBS) $(LIBPNG_LIBS) $(ZLIB_LIBS)
+LDADD = $(KPATHSEA_LIBS) $(FREETYPE2_LIBS) $(LIBPNG_LIBS) $(ZLIB_LIBS) -lpaper
 
 if build_MacOsX
 AM_LDFLAGS = -framework ApplicationServices
